es-es/c++-es.html.markdown

```diff
@@ -8,6 +8,7 @@ contributors:
     - ["Connor Waters", "http://github.com/connorwaters"]
 translators:
     - ["Gerson Lázaro", "https://gersonlazaro.com"]
+    - ["Diego Ramirez", "https://DiddiLeija.github.io"]
```

Comment:
I think my job is a "translation" work, because I've been fixing grammar mistakes, and things like that.